src/pages/Sketch/components/Canvas.jsx

```diff
@@ -493,27 +498,45 @@ function Canvas() {
       });
   };
 
-  const initiateLoadSaved = () => document.getElementById("file-selector").click();
-  const loadLastState = e => {
+
+  
+ 
+  const initiateload = () => document.getElementById("file-selector").click();
+  const loadimport= e =>{   
     let file = e.target.files[0];
-    if (!file) return;
-    let reader = new FileReader();
+    var textFile = /image.*/;
+
+
+    if (!(file.type.match(textFile))) { 
+     alert("Sorry, We cannot import the selected file . File should be of type jpg/jpeg/png/webp");
```

Comment:
Make this.

`OOPs! You didn't select an image. We only support jpg,png and webp'

Reply:
ohk